src/jrd/extds/ExtDS.h

```diff
@@ -197,9 +197,6 @@ class Provider : public Firebird::GlobalStorage
 };
 
 // Provider flags
-const int prvMultyStmts		= 0x0001;	// supports many active statements per connection
-const int prvMultyTrans		= 0x0002;	// supports many active transactions per connection
-const int prvNamedParams	= 0x0004;	// supports named parameters
 const int prvTrustedAuth	= 0x0008;	// supports trusted authentication
 
```

Comment:
prvTrustedAuth should be re-numbered as 0x0001, I suppose.

Reply:
Actually, it should be removed at all because whether remote side support trusted auth or not is a matter of config, but I'll renumber it.